design-editor/src/panel/preview/preview-element.js

```diff
@@ -4,7 +4,9 @@ import {StateManager} from '../../system/state-manager';
 import {DressElement} from '../../utils/dress-element';
 import {EVENTS, eventEmitter} from '../../events-emitter';
 import pathUtils from '../../utils/path-utils';
+import utils, {checkGlobalContext} from '../../utils/utils';
```

Comment:
utils not needed?

Reply:
It was accidentally added by VSCode Editor, and now it's removed